buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -630,6 +631,7 @@ class BeamModulePlugin implements Plugin<Project> {
         guava_testlib                               : "com.google.guava:guava-testlib:$guava_version",
         hadoop_client                               : "org.apache.hadoop:hadoop-client:$hadoop_version",
         hadoop_common                               : "org.apache.hadoop:hadoop-common:$hadoop_version",
+        hadoop_mapred                               : "org.apache.hadoop:hadoop-mapred:0.22.0",
```

Comment:
should this be $hadoop_version instead of a hardcoded value?

Reply:
Thanks for noticing, I removed that dependency.
We can use `hadoop_mapreduce_client_core` instead.